.travis.yml

```diff
@@ -4,16 +4,21 @@ services:
   - postgresql
 matrix:
   include:
-    - { python: "2.7",   env: "TOXENV=py-full DATABASE_URL=postgres://localhost/travis_ci_test" }
-    - { python: "3.5",   env: "TOXENV=py-full DATABASE_URL=postgres://localhost/travis_ci_test" }
-    - { python: "3.6",   env: "TOXENV=py-full DATABASE_URL=postgres://localhost/travis_ci_test" }
-    - { python: "3.7",   env: "TOXENV=py-full DATABASE_URL=postgres://localhost/travis_ci_test" }
+    - { python: "2.7",   env: "TOXENV=py-full-marshmallow2 DATABASE_URL=postgres://localhost/travis_ci_test" }
+    - { python: "3.5",   env: "TOXENV=py-full-marshmallow2 DATABASE_URL=postgres://localhost/travis_ci_test" }
+    - { python: "3.6",   env: "TOXENV=py-full-marshmallow2 DATABASE_URL=postgres://localhost/travis_ci_test" }
+    - { python: "3.7",   env: "TOXENV=py-full-marshmallow2 DATABASE_URL=postgres://localhost/travis_ci_test" }
 
-    - { python: "2.7",   env: "TOXENV=py-base" }
-    - { python: "3.6",   env: "TOXENV=py-base" }
+    - { python: "2.7",   env: "TOXENV=py-base-marshmallow2" }
+    - { python: "3.6",   env: "TOXENV=py-base-marshmallow2" }
 
-    - { python: "pypy2.7-6.0",  env: "TOXENV=py-full" }
-    - { python: "pypy3.5-6.0", env: "TOXENV=py-full" }
+    - { python: "2.7",   env: "TOXENV=py-full-marshmallow3" }
+    - { python: "3.6",   env: "TOXENV=py-full-marshmallow3" }
+    - { python: "2.7",   env: "TOXENV=py-base-marshmallow3" }
+    - { python: "3.6",   env: "TOXENV=py-base-marshmallow3" }
```

Comment:
This is getting to be a lot of jobs. Perhaps we remove the pypy jobs? Since this is a pure-python package, there's no reason this shouldn't work on pypy if it works on 2.7 and 3.5.